pages/common/dot.md

```diff
@@ -1,12 +1,25 @@
 # dot
 
-> A command-line tool to produce layered drawings of directed graphs.
-> More information: <https://www.graphviz.org/pdf/dotguide.pdf>.
+> Render an image of a _linear directed_ network graph from a `graphviz` file.
+> Layouts are `dot`, `neato`, `twopi`, `circo`, `fdp`, `sfdp`, `osage` & `patchwork`.
+> More information: <http://graphviz.org/doc/info/command.html>.
 
-- Render an image file and determine output filename based on input filename and selected format:
+- Render a 'png' image with a filename based on the input filename and output format:
 
-`dot -Tpng -O {{path/to/file.dot}}`
+`dot -Tpng -O {{path/to/input.gv}}`
 
-- Create an SVG from DOT file:
+- Render a `svg` image with the specified output filename:
 
-`dot -Tsvg -o {{path/to/out_file.svg}} {{path/to/file.dot}}`
+`dot -Tsvg -o {{path/to/image.svg}} {{path/to/input.gv}}`
+
+- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
```

Comment:
Wouldn't this example alone be enough to demonstrate the `-T{{format}}` flag?

Reply:
Disagree. Yes, it demonstrates the -Tfmt flag but there is not enough room on a line to also explain the -o and -O flags that are introduced on the lines above. Also, I was preserving the previously accepted examples when I added this multi-format example.

Reply:
Oh, I didn't notice the `o` vs `O`. This is really evil.